payments_service/auth/views.py

```diff
@@ -0,0 +1,42 @@
+import logging
+
+from django.core.exceptions import ObjectDoesNotExist
+
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+from slumber.exceptions import HttpClientError
+
+from .. import solitude
+from ..base.views import error_400
+from .forms import SignInForm
+
+log = logging.getLogger(__name__)
+
+
+@api_view(['POST'])
+def sign_in(request):
+    form = SignInForm(request.DATA)
+
+    if not form.is_valid():
+        return error_400(response=form.errors)
+
+    fxa_uuid = u'fxa:{u}'.format(u=form.cleaned_data['access_token']['user'])
```

Comment:
Thanks for that reminder. We should file a bug to switch new users in webpay over to this scheme as well, so that we'll all be synced up at some point.